mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.tmpl

```diff
@@ -1253,7 +1253,7 @@ func TestAccComputeInstance_serviceAccount(t *testing.T) {
 						"https://www.googleapis.com/auth/userinfo.email"),
 				),
 			},
-			computeInstanceImportStep("us-central1-a", instanceName, []string{}),
```

Comment:
Could you help me understand why we are changing the tests? This tends to imply a change in behavior which is not what we want for this resource.

Reply:
When we import an instance during a test and do not create a new one, we need to add an allow_stopping_for_update resource. And IMHO this is the correct behaviour - we have an existing instance that has some configuration and we want to change some resources, which requires us to stop the instance.
In the case of testAccComputeInstance_standardVM_maxRunDurationUpdated, there was no import of the instance, but the max_run_duration for the existing instance was changed, which also requires the instance to be stopped, so the test configuration should include allow_stopping_for_update.

```
"Changing the machine_type, min_cpu_platform, service_account, enable_display, shielded_instance_config, scheduling.node_affinities, scheduling.max_run_duration " +
"or network_interface.[#d].(network/subnetwork/subnetwork_project) or advanced_machine_features on a started instance requires stopping it. "
```